package.json

```diff
@@ -13,14 +13,16 @@
         "serve": "vue-cli-service serve",
         "build": "vue-cli-service build && shx rm -rf ./docs && shx cp -r ./dist ./docs",
         "test": "vue-cli-service test:unit --coverage && npm run test:cypress",
+        "test-component": "vue-cli-service test:unit --coverage && cd dist && live-server --port=8080 --entry-file=./demo.html & npm run test:cypress",
         "test:unit": "vue-cli-service test:unit",
+        "build-component": "rm -rf dist && vue-cli-service build --target wc --name license-chooser src/main.js",
         "test:cypress": "node_modules/.bin/cypress open",
         "lint": "vue-cli-service lint src/**/*.{vue,js} --fix && npx stylelint ./src/**/*.{vue,css} --fix",
         "i18n:report": "vue-cli-service i18n:report -v --src src/**/*.vue --locales src/locales/**.json -o output.json"
     },
     "dependencies": {
         "@creativecommons/vocabulary": "^2020.11.3",
-        "@creativecommons/vue-vocabulary": "^2020.11.3",
+        "@creativecommons/vocabulary-components": "^0.1.0",
```

Comment:
Can we also replace `@creativecommons/vocabulary` with `@creativecommons/vocabulary-styles` as part of this PR?

Reply:
Can not remove the `@creativecommons/vocabulary` dependency here due to the following scss imports

1. In StepNavigation.vue: color.scss
2. And in VInput.vue
  - color.scss
  - typography.scss
3. Also using fonts from this lib

<img width="465" alt="Screenshot 2022-01-15 at 3 07 44 PM" src="https://user-images.githubusercontent.com/10625985/149617227-060f008a-b9b3-4d56-8fb1-ac0116d5f97d.png">

Reply:
OK, we will include the SCSS files in the next `vocabulary-styles` release.

Reply:
We will likely make the new `vocabulary-styles` release today.

Reply:
https://github.com/creativecommons/vocabulary-styles/pull/4

Reply:
`vocabulary-styles` 0.2.0 was just released, which includes the SASS files:
https://www.npmjs.com/package/@creativecommons/vocabulary-styles